sorc/chgres_cube.fd/atm_input_data.F90

```diff
@@ -0,0 +1,2844 @@
+!> @file
+!! @brief Read atmospheric and surface data from GRIB2, NEMSIO and NetCDF files.
+!! @author George Gayno, Kyle Gerheiser NCEP/EMC
+
+!> Read atmospheric data on the input grid.
+!! Supported formats include fv3 tiled 'restart' files, fv3 tiled 
+!! 'history' files, fv3 gaussian history files, spectral gfs
+!! gaussian nemsio files, and spectral gfs sigio/sfcio files.
+!!
+!! Public variables are defined below: "input" indicates field
+!! associated with the input grid.
+!!
+!! @author George Gayno, Kyle Gerheiser NCEP/EMC
+module atm_input_data_mod
+  use esmf
+  use netcdf
+  use nemsio_module
+
+
+  use program_setup, only          : data_dir_input_grid, &
```

Comment:
Check the 'use' statements in the new modules. Most of these variables are not used.

Reply:
Ah, I meant to do that